crates/mockiato-codegen/src/generate/visibility.rs

```diff
@@ -0,0 +1,33 @@
+use syn::{parse_quote, Visibility};
+
+pub(super) fn raise_visibility_by_one_level(visibility: &Visibility) -> Visibility {
+    match visibility {
+        Visibility::Public(_) | Visibility::Crate(_) => visibility.clone(),
+        Visibility::Inherited => parse_quote!(pub(super)),
+        Visibility::Restricted(inner_visibility)
+            if inner_visibility.path.segments[0].ident == "crate" =>
+        {
+            visibility.clone()
+        }
+        Visibility::Restricted(inner_visibility)
+            if inner_visibility.path.segments[0].ident == "self" =>
+        {
+            let mut inner_visibility = inner_visibility.clone();
+            inner_visibility.path.segments[0].ident = parse_quote!(super);
```

Comment:
`inner_visibility.path.segments[0].ident` is repeated a few times, put it into a variable.

Reply:
Addressed in 2646ae2